components/list/test/list-nested.visual-diff.js

```diff
@@ -0,0 +1,45 @@
+import puppeteer from 'puppeteer';
+import { VisualDiff } from '@brightspace-ui/visual-diff';
+
+describe('d2l-list-nested', () => {
+
+	const visualDiff = new VisualDiff('list-nested', import.meta.url);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser, { viewport: { width: 1000, height: 8500 } });
+	});
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+		await page.mouse.move(-1, -1);
```

Comment:
Curious what the mouse move was needed for?

Reply:
Looks like it is from here: https://github.com/BrightspaceUI/core/pull/3434

From Erik:
>I also had to move the mouse because tests that use hover() move the mouse on the screen and then leave it there, which was causing hover styles to appear on this list!

Reply:
Seeing as these particular tests don't hover then it should be safe to remove.

This is good though -- I've added resetting the mouse position to my reset logic in the new visual diff stuff.